src/pages/Bonding/Bonding.tsx

```diff
@@ -24,11 +24,15 @@ export const Bonding: React.FC = () => {
   const navigate = useNavigate();
 
   const checkIfUserIsAdmin = () => {
+    const splittedAddresses = bondContractAdminDevnet.split(", ");
```

Comment:
Here it should be a bit less restrictive.

Just split by "," and then use trim() to get rid of white spaces